augerctl/command/printer.go

```diff
@@ -34,6 +34,8 @@ func NewPrinter(w io.Writer, printerType string) Printer {
 	switch printerType {
 	case "yaml":
 		return &yamlPrinter{w: w}
+	case "json":
+		return &jsonPrinter{w: w}
```

Comment:
Just want to check is it intentional that the output includes a `---`  and comment at the start?  I tend to think of json output as more machine parse-able format and I couldn't see any flag to just output raw json only.

```bash
root@02f91f29d99e:~/auger/augerctl# ./augerctl get services -n default -o json
---
# /registry/services/specs/default/kubernetes | application/vnd.kubernetes.storagebinary
{"kind":"Service","apiVersion":"v1","metadata":{"name":"kubernetes","namespace":"default","uid":"90017205-287e-4431-a462-9557ac3e4844","creationTimestamp":"2024-08-24T09:23:41Z","labels":{"component":"apiserver","provider":"kubernetes"},"managedFields":[{"manager":"kube-apiserver","operation":"Update","apiVersion":"v1","time":"2024-08-24T09:23:41Z","fieldsType":"FieldsV1","fieldsV1":{"f:metadata":{"f:labels":{".":{},"f:component":{},"f:provider":{}}},"f:spec":{"f:clusterIP":{},"f:internalTrafficPolicy":{},"f:ipFamilyPolicy":{},"f:ports":{".":{},"k:{\"port\":443,\"protocol\":\"TCP\"}":{".":{},"f:name":{},"f:port":{},"f:protocol":{},"f:targetPort":{}}},"f:sessionAffinity":{},"f:type":{}}}}]},"spec":{"ports":[{"name":"https","protocol":"TCP","port":443,"targetPort":32765}],"clusterIP":"10.0.0.1","clusterIPs":["10.0.0.1"],"type":"ClusterIP","sessionAffinity":"None","ipFamilies":["IPv4"],"ipFamilyPolicy":"SingleStack","internalTrafficPolicy":"Cluster"},"status":{"loadBalancer":{}}}

```